test/Ssh.Tests.ps1

```diff
@@ -2,11 +2,14 @@
 
 Describe 'SSH Function Tests' {
     Context 'Get-SshPath Tests' {
+        BeforeAll {
+            $sepChar = [System.IO.Path]::DirectorySeparatorChar
+        }
         It 'Returns the correct default path' {
-            Get-SshPath | Should BeExactly $Home\.ssh\id_rsa
+            Get-SshPath | Should BeExactly "${Home}${sepChar}.ssh${sepChar}id_rsa"
         }
         It 'Returns the correct path given a filename' {
-            Get-SshPath mykey | Should BeExactly $Home\.ssh\mykey
+            Get-SshPath mykey | Should BeExactly "${Home}${sepChar}.ssh${sepChar}mykey"

```

Comment:
This works, but is kind of gross. I wonder if Pester could learn `Should BePath` that would translate `/` in the expected value to `[System.IO.Path]::DirectorySeparatorChar`?